OrbitGl/App.h

```diff
@@ -233,6 +233,14 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
 
   DataView* GetOrCreateDataView(DataViewType type) override;
 
+  const std::unique_ptr<ProcessManager>& GetProcessManager() {
```

Comment:
Also should this be `nodiscard`, since the other methods that you added are also `nodiscard`?

Reply:
Trust me, I feel you...